website/blog/2025-05-22-release-1.19.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Podman Desktop 1.19 Release
+description: Podman Desktop 1.19 has been released!
+slug: podman-desktop-release-1.19
+authors: [axel7083]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.19.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.19 Release! 🎉
+
+![podman-desktop-hero-1.19](/img/blog/podman-desktop-release-1.19/banner.png)
+
+Podman Desktop 1.19 is now available. [Click here to download it](/downloads)!
+
+This release among many extension update brings exciting new features and improvements:
+
+- **Podman v5.5**: The Podman executable shipped with Podman Desktop has been updated to its latest version.
```

Comment:
Should we call out the fix with the startup of podman machine on Mac? or something new valuable from podman 5.5? https://github.com/containers/podman/releases 
I think we could call out the support for artifact which is getting better?

Reply:
I would not go too much into podman details, as this is not something supported in Podman Desktop..